src/google/protobuf/json/internal/zero_copy_buffered_stream.h

```diff
@@ -67,24 +67,24 @@ class MaybeOwnedString {
 
   // Returns the string as a view, regardless of whether it is owned or not.
   absl::string_view AsView() const {
-    if (auto* unowned = absl::get_if<StreamOwned>(&data_)) {
+    if (auto* unowned = std::get_if<StreamOwned>(&data_)) {
       return unowned->AsView();
     }
 
-    return absl::get<std::string>(data_);
+    return std::get<std::string>(data_);
```

Comment:
You're getting build errors here in some configurations, missing include maybe?

Reply:
Fixed